cloud_sql/mysql_instance_backupdr_backup/main.tf

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# [START cloud_sql_mysql_instance_backupdr_backup]
+resource "google_sql_database_instance" "instance" {
+  name             = "mysql-instance-backupdr-backup"
+  region           = "asia-northeast1"
+  database_version = "MYSQL_8_0"
+  settings {
+    tier = "db-f1-micro"
+    backup_configuration {
+      enabled            = true
+      binary_log_enabled = true
+      start_time         = "20:55"
+    }
+  }
+  backupdr_backup = "projects/<project-id>/locations/asia-northeast1/backupVaults/<backup-vault-id>/dataSources/<data-source-id>/backups/<backup-id>"
```

Comment:
I understand that this sample is not being actively tested in CI, but is this the correct form of this backup identifier? Normally if we need values like this, we reference the identifiers of other resources. 

Are you able to use something like the google_backup_dr_backup data source to reference this value?  It's unclear if this is the exact mapping data source, but something like it should be used instead of a placeholder value.